tests/main/interfaces-cifs-mount/task.yaml

```diff
@@ -0,0 +1,70 @@
+summary: Ensure that the cifs-mount interface works.
+
+details: |
+    The cifs-mount interface allows mounting and unmounting CIFS filesystems.
+
+prepare: |
+    snap install test-snapd-cifs-mount
+
+    # Create linux and samba users 
+    sudo useradd -s /bin/true test-cifs
+    echo "test-cifs:test123#" | sudo chpasswd
+    mkdir /home/test-cifs
+
+    echo test > test-pass
+    echo test >> test-pass
+    sudo smbpasswd -a test-cifs -s < test-pass
+
+    # Create mount point
+    mkdir -p local_share /var/snap/test-snapd-cifs-mount/current/local_mount
+    chmod 755 local_share
+    touch local_share/test
+
+    # Configure samba
+    cp /etc/samba/smb.conf /etc/samba/smb.conf.back
+    echo "[data]" >> /etc/samba/smb.conf
+    echo "path=$(pwd)/local_share" >> /etc/samba/smb.conf
+    echo "writable=yes" >> /etc/samba/smb.conf
+    echo "browsable=yes" >> /etc/samba/smb.conf
+
+    systemctl restart smbd.service
+    systemctl restart nmbd.service
```

Comment:
You don't need to touch `nmbd`, IIRC it's only used for NetBIOS name resolution and shouldn't matter here.